### PR TITLE
Daily

### PR DESCRIPTION
**Pull request by elguapogordo**
added daily horrorscope page
placeholders for content
added initial spec test
added route
added daily controller with index action

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -42,14 +42,14 @@ gem 'figaro'
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
-  gem 'rspec-rails'
   gem 'capybara'
+  gem 'factory_bot_rails'
+  gem 'faker'
+  gem 'launchy'
```
Comment (elguapogordo) [inline Gemfile:48]: adds save_and_open_page function

**Comment by madhalle** [comment]
Looks good!!

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -87,3 +83,23 @@ p{
...
+  text-align: center;
+  padding-top: 30px;
+  letter-spacing: 5px;
+}
+
+.daily-subtitle {
+  font-size: 12px;
+  letter-spacing: 9px;
+  text-align: center;
+}
```
Comment (elguapogordo) [inline app/assets/stylesheets/application.css:105]: css with page specific settings for daily page, but still following welcome page formatting

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -54,6 +54,7 @@
       <div class='navbar'>
         <ul>
           <li style="float:left"><a class= "nav-button1"><%= link_to 'HOME', "/"%></a></li>
+          <li style="float:left"><a class= "nav-button1"><%= link_to 'DAILY', daily_index_path %></a></li>
```
Comment (elguapogordo) [inline app/views/layouts/application.html.erb:57]: link to daily_index_path added to navbar

File: spec/features/visitors/daily_spec.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+RSpec.describe 'visitor can visit daily horrorscope page' do
+  it 'and see what possibilities await' do
+    visit '/daily'
+
+    expect(page).to have_content('Daily Darkness')
+    expect(page).to have_content('Four Harbingers')
+    save_and_open_page
+  end
+end
```
Comment (elguapogordo) [inline spec/features/visitors/daily_spec.rb:11]: more tests to come as content and microservice are added

**Pull request merged**